freqtrade/strategy/strategy_helper.py

```diff
@@ -58,7 +58,10 @@ def merge_informative_pair(dataframe: pd.DataFrame, informative: pd.DataFrame,
     return dataframe
 
 
-def stoploss_from_open(open_relative_stop: float, current_profit: float) -> float:
+def stoploss_from_open(
+    open_relative_stop: float,
+    current_profit: float
+) -> float:
```

Comment:
I'm not sure on the point of these formatting changes.

I found it easier to read in one line (especially considering that this line is not that long).

Reply:
I remember now, I had to add a parameter for shorting which made it over 100 characters long